src/main/java/core/basesyntax/service/operations/report/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.operations.report.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.operations.report.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String SPLITTER_CSV = ",";
+    private static final String HEADER_CSV = "fruit,quantity";
+    private static final String SPLITTER_NEW_LINE = System.lineSeparator();
+
+    public String generateReportContent(Map<Fruit, Integer> storage) {
```

Comment:
it would make sense to have a field of type `Storage` and call `storage.getFruits()` on it by itself here instead of passing the map as a parameter, because getting the data from the DB sounds like a part of this functionality and thus a component of the `ReportGenerator`